### PR TITLE
Three NewCrop eRx fixes

### PR DESCRIPTION
**Pull request by kchapple**
1- Func required two params in eRx Builder 2- Fixed javascript XSS security error when tabs attempted to reach across domain and read newcrop frame. 3- Fixed tab so it displays New Crop rather than Unknown. I believe this was already merged into 501 rel for patch 7.

### CODE REVIEW

File: interface/eRx.php [hunk 1]
```diff
@@ -48,6 +48,9 @@ function array_key_exists_default($key, $search, $default = null)
 
 ?>
 <html>
+    <head>
+        <title><?php echo xl('New Crop'); ?></title>
```
Comment (bradymiller) [inline interface/eRx.php]: xlt:
```
<title><?php echo xlt('New Crop'); ?></title>
```

**Comment by bradymiller** [comment]
Need to increment the following value by 1 (cache busting mechanism) since modifying js scripts:
https://github.com/openemr/openemr/blob/master/version.php#L50

**Comment by bradymiller** [comment]
hi @kchapple ,
Review complete and only needs 2 very minor issues to be addressed. Then will be ready for codebase. thanks! -brady

**Comment by bradymiller** [comment]
hi @kchapple ,
To keep the momentum going, I'm gonna bring this in and make the minimal fixes in a proceeding commit. Very happy to do this and thanks for the fix! -brady

**Comment by kchapple** [comment]
Sorry I dropped the ball on this. 

Ken

**Comment by bradymiller** [comment]
no prob and thanks for the fix! -brady

**Pull request merged**